packages/target-tools/src/telemetryProvider.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/prefer-default-export */
+
+import { now } from "./lodash";
+import { noop } from "./utils";
+import { DECISIONING_METHOD } from "./enums";
+
+/**
+ * The get TelemetryProvider initialization method
+ * @param {function} sendTelemetriesFunc function used to send the telemetries, required
+ */
+export function TelemetryProvider(
+  request,
+  executeTelemetriesFunc = noop,
```

Comment:
I think we should remove `noop` as the default value for `executeTelemetriesFunc`, and just make it a required arg w/ no default